### DISCUSSION TITLE
How could I not display past events in shortcode listing

### DISCUSSION THREAD

**Discussion by 1JaWi** [wpeventmanager/wp-event-manager#1362] [Q&A]
Past events should not show up in the list, how could I fix it?
Code: [events per_page="4" layout_type="list" order="ASC" show_filters="false" orderby="event_start_date" show_event_types="false" show_tags="false" show_pagination="true" Cancelled="false"]

**Comment by DevByDoll** [comment]
Show under settings event list 
Hide Expired listings
